Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ghcr.io/tiiuae/fog-ros-baseimage:sha-f70aceb
```

Comment:
This base image is still foxy, not galactic. Will that cause trouble?

Reply:
Yes. This will be a problem at least for this component because it does not build for foxy anymore.